_pkgdown.yml

```diff
@@ -14,50 +19,26 @@ template:
       creator: "@rstudio"
       card: summary
 
-development:
-  version_tooltip: "Development version"
```

Comment:
We added that explicitly to show that this is a doc for the dev version always. I know the 3 versions digits is quite specific. The mode `auto` won't work obviously with the version schemes. Also we don't have doc for the release version only - so using `mode: unreleased` was a bit too much with the danger badge/color added. 

With the new `PKGDOWN_DEV_MODE` env var I could build two sites now probably to recreate the mode auto using GHA. But I remember having two websites was something Yihui's was found off.

Reply:
Oh I thought this was to work around a different buglet that I fixed. I can revert it.